types/searchResult.ts

```diff
@@ -0,0 +1,9 @@
+export interface SearchResult {
```

Comment:
is this and the file below it basically the equivalent to the `models` that Ignacio made? and therefore should these files be removed?